debian/libostree-1-1.symbols

```diff
@@ -26,6 +26,7 @@ libostree-1.so.1 libostree-1-1 #MINVER#
  LIBOSTREE_2018.3@LIBOSTREE_2018.3 2018.3
  LIBOSTREE_2018.5@LIBOSTREE_2018.5 2018.5
  LIBOSTREE_2018.6@LIBOSTREE_2018.6 2018.6
+ LIBOSTREE_2018.9@LIBOSTREE_2018.9 2018.9
```

Comment:
The package version (the second argument) should match what you used for the symbol - 2018.6-3endless2. When we rebase we'll reset both to 2018.9.

Reply:
Ok. Corrected the typo. Pushed again.